person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = 111
```

Comment:
- [ ] You have done nice work overall. I think id should not be a constant value. You can use a random number generator here. Kindly make changes. :pray: